optuna/integration/pytorch_lightning.py

```diff
@@ -40,6 +40,12 @@ class PyTorchLightningPruningCallback(Callback):
             ``pytorch_lightning.LightningModule.training_step`` or
             ``pytorch_lightning.LightningModule.validation_epoch_end`` and the names thus depend on
             how this dictionary is formatted.
+
+    .. note::
+        For the distributed data parallel training, ``accelerator="ddp"``,
```

Comment:
[Question] PyTorch Lightning supports multiple accelerators (e.g., `dp`, `ddp_spawn`, `ddp2`, and `horovod`) in addition to `ddp`. Does the callback support them?
https://pytorch-lightning.readthedocs.io/en/stable/advanced/multi_gpu.html#distributed-modes

Reply:
I tested https://github.com/optuna/optuna-examples/pull/43 only in CPU environment and multi GPUs environment(2GPUs on one machine).
In multi GPUs envirioment, if accelerator is not specified in pl.Trainer, accelerator="ddp_spawn" is set.
https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/trainer/connectors/accelerator_connector.py#L752-L757
So I tested only in ddp_spawn and ddp_cpu.